### PR TITLE
manage order

### PR DESCRIPTION
**Pull request by quyensama**
## Related Tickets:
- [#45909](https://edu-redmine.sun-asterisk.vn/issues/45909)

## WHAT this PR does?
- #45945: [Admin][Order] View list order
- #45948: [Admin][Order] Edit/ Change Order Status
## Notes Impacted Areas:

- N/A.

## Attachments:
Screenshot from 2022-03-07 12-23-27

Screenshot from 2022-03-07 11-17-52

Screenshot from 2022-03-07 11-12-43

### CODE REVIEW

**Review by NamBZ — APPROVED** [review]

**Comment by quyensama** [comment]
ready

File: resources/views/dashboards/admin/orders/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                                            <td>
+                                                @if ($order->status==0)
+                                                    <span class="text text-primary">{{ __('New Order') }}</span>
+                                                @elseif ($order->status == 1)
+                                                    <span class="text text-primary">{{ __('In Proccess') }}</span>
+                                                @elseif ($order->status == 2)
+                                                    <span class="text text-primary">{{ __('In Shipping') }}</span>
+                                                @elseif ($order->status == 3)
+                                                    <span class="text text-success">{{ __('Delivery Completed') }}</span>
+                                                @elseif ($order->status == 4)
```
Comment (ghost) [inline resources/views/dashboards/admin/orders/index.blade.php]: bỏ số vào config nha

File: resources/views/dashboards/admin/orders/viewOrder.blade.php [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                <div class="card">
+                    <div class="card-header bg bg-info">
+                        <h3 class="card-title col-md-12 text-center">{{ __('Customer Info') }}</h3>
+                    </div>
+                    <!-- /.card-header -->
+                    <div class="card-body">
+                        <table class="table table-bordered">
+                            <thead>
+                                <tr>
+                                    <th style="width: 10px">#</th>
```
Comment (ghost) [inline resources/views/dashboards/admin/orders/viewOrder.blade.php]: bỏ style ra file css riêng

**Comment by quyensama** [comment]
ready

**Pull request merged**